server/models/user.js

```diff
@@ -0,0 +1,12 @@
+'use strict';
+module.exports = (sequelize, DataTypes) => {
+  const User = sequelize.define('User', {
+    firstName: DataTypes.STRING,
+    lastName: DataTypes.STRING,
+    email: DataTypes.STRING
```

Comment:
I noticed that there is createdAt and updatedAt in migrations and not here - do you not need access to them

Reply:
So I played around with the **sequelize-cli** today 
Even though we don't have an createdAt & updatedAt within our mock model, the **sequelize-cli** will generate a createdAt & updatedAt column within our postgresql database.

Anytime we create a **User** in this case, those fields will auto-populate. 
I'm not sure if this answers this comment, but we do have access to them. 

But if best practice is to include the createdAt and updatedAt within the model, we can add a **timestamp** option field each time we create the model using **sequelize-cli**. (But I'm not sure if this will affect it in anyway, something we can test out :smile:)

Reply:
Yeah, it might not be totally necessary to include createdAt and updatedAt in the user schema.

I've added new fields in user.js anyways in the latest commit but its always easy to go back and update them.